lib/active_decorator/railtie.rb

```diff
@@ -33,6 +33,9 @@ class Railtie < ::Rails::Railtie
         require 'active_decorator/monkey/active_record/associations'
         ActiveRecord::Associations::Association.send :prepend, ActiveDecorator::Monkey::ActiveRecord::Associations::Association
         ActiveRecord::Associations::CollectionAssociation.send :prepend, ActiveDecorator::Monkey::ActiveRecord::Associations::CollectionAssociation
+        if Rails.version.to_f >= 4.0
```

Comment:
`#take` was added in 4.0.0 https://github.com/rails/rails/commit/1379375f93c53d4c49fa8592b6117c3ade263f2e